lib/inventory_refresh/application_record_iterator.rb

```diff
@@ -2,19 +2,11 @@ module InventoryRefresh
   class ApplicationRecordIterator
     attr_reader :inventory_collection, :manager_uuids_set, :iterator, :query
```

Comment:
@Ladas can `@manager_uuids_set` be deleted now?

Reply:
Actually are any of these ivars still used?

Reply:
oh yeah

Reply:
done